docs/demos/onvif-authentication-access.md

```diff
@@ -0,0 +1,306 @@
+### Passing credentials to Onvif Discovery Handler to discover authenticated devices
+Make sure you have at least one Onvif camera that is reachable so Onvif discovery handler can discovery your Onvif camera. To test accessing Onvif with credentials, make sure your Onvif camera is authentication-enabled. **Write down the username and password**, they are required in the flow below.
+
+#### Acquire Onvif camera's device uuid
+
+First use the following helm chart to deploy an Akri Configuration that uses ONVIF discovery handler and see if your camera is discovered.
+
+```bash
+# add akri helm charts repo
+helm repo add akri-helm-charts https://project-akri.github.io/akri/
+# ensure helm repos are up-to-date
+helm repo update
+```
+Set up the Kubernetes distribution being used, here we use 'k8s', make sure to replace it with a value that matches the Kubernetes distribution you used.
+
+See [the cluster setup steps](../user-guide/cluster-setup.md#configure-crictl) for information on how to set the crictl configuration variable `AKRI_HELM_CRICTL_CONFIGURATION`
+
+ ```bash
+export AKRI_HELM_CRICTL_CONFIGURATION="--set kubernetesDistro=k8s"
+```
```

Comment:
Put this in a separate section before the current one, so the current one can be safely skipped if the user already have the camera UUID

Reply:
split the paragraph into two.